src/test/java/org/folio/rest/api/ReferenceTablesTest.java

```diff
@@ -497,7 +508,7 @@ public void instanceStatusesBasicCrud()
     assertThat(postResponse.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
 
     String entityUUID = postResponse.getJson().getString("id");
-    URL entityUrl = instanceStatusesUrl("/" + entityUUID);
+    instanceStatusesUrl("/" + entityUUID);
```

Comment:
Is this call needed anymore, given the assignment is no longer done? Does it have any side effects? 

If not, I think it can be removed

Reply:
Nope. Removed.